internal/routes/server.go

```diff
@@ -438,6 +438,9 @@ func statsRoutes(r *mux.Router, c *stats.Controller) {
 
 	r.Handle("/realm/key-server.csv", c.HandleKeyServerStats(stats.TypeCSV)).Methods(http.MethodGet)
 	r.Handle("/realm/key-server.json", c.HandleKeyServerStats(stats.TypeJSON)).Methods(http.MethodGet)
+
+	r.Handle("/composite.csv", c.HandleComposite(stats.TypeCSV)).Methods(http.MethodGet)
```

Comment:
This should still be nested under `realm`. I want to future-proof a stats API that is for the system.